app/assets/javascripts/discourse/models/user.js.es6

```diff
@@ -611,6 +611,20 @@ const User = RestModel.extend({
     }
   },
 
+  ignore() {
+    return ajax(`${userPath(this.get("username"))}/ignore.json`, {
+      type: "PUT",
+      data: { ignored_user_id: this.get("id") }
+    });
+  },
+
+  watch: function() {
+    return ajax(`${userPath(this.get("username"))}/watch.json`, {
+      type: "DELETE",
```

Comment:
You changed the verb, but not the endpoint ;)

Reply:
Yes 🤒